### PR TITLE
docs: fix word spellings

### PR DESCRIPTION
**Pull request by njzjz**

### CODE REVIEW

File: deepmd/tf/nvnmd/entrypoints/wrap.py
Comment (njzjz) [inline deepmd/tf/nvnmd/entrypoints/wrap.py:1]: should not changed

File: doc/model/dplr.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ pair_style      deepmd ener.pb
 pair_coeff      * *
 bond_style      zero
 bond_coeff      *
-special_bonds   lj/coul 1 1 1 angle no
+special_bonds   lj/could 1 1 1 angle no
```
Comment (njzjz) [inline doc/model/dplr.md:171]: reverted

File: examples/data_conv/OUTCAR
Comment (njzjz) [inline examples/data_conv/OUTCAR:1]: Revert

File: examples/water/dplr/lmp/in.lammps
Comment (njzjz) [inline examples/water/dplr/lmp/in.lammps:1]: Revert

File: source/api_cc/src/DeepTensorTF.cc [hunk 2]
```diff
@@ -65,10 +65,10 @@ void DeepTensorTF::init(const std::string &model,
   }
   if (!model_compatable(model_version)) {
     throw deepmd::deepmd_exception(
-        "incompatable model: version " + model_version +
+        "incompatible model: version " + model_version +
         " in graph, but version " + global_model_version +
         " supported "
-        "See https://deepmd.rtfd.io/compatability/ for details.");
+        "See https://deepmd.rtfd.io/compatibility/ for details.");
```
Comment (njzjz) [inline source/api_cc/src/DeepTensorTF.cc:71]: Need to update the url

File: source/lmp/tests/test_dplr.py
Comment (njzjz) [inline source/lmp/tests/test_dplr.py:1]: Revert

File: source/tests/pt/model/test_dpa1.py
Comment (njzjz) [inline source/tests/pt/model/test_dpa1.py:1]: Revert

Comment (njzjz) [inline source/tests/pt/model/test_dpa1.py:1]: Revert

File: source/tests/pt/model/test_dpa2.py
Comment (njzjz) [inline source/tests/pt/model/test_dpa2.py:1]: Revert

File: source/tests/pt/model/test_se_atten_v2.py
Comment (njzjz) [inline source/tests/pt/model/test_se_atten_v2.py:1]: Revert

File: source/tests/tf/test_data_large_batch.py
Comment (njzjz) [inline source/tests/tf/test_data_large_batch.py:1]: Revert

File: source/tests/tf/test_descrpt_hybrid.py
Comment (njzjz) [inline source/tests/tf/test_descrpt_hybrid.py:1]: Revert

File: source/tests/tf/test_type_one_side.py
Comment (njzjz) [inline source/tests/tf/test_type_one_side.py:1]: Revert

File: source/api_cc/src/DeepTensorTF.cc (hunk 2 above)
Comment (njzjz) [reply source/api_cc/src/DeepTensorTF.cc:71]: Done: https://deepmd.rtfd.io/compatibility/

**Review by wanghan-iapcm — APPROVED** [review]

File: deepmd/pt/model/descriptor/se_r.py [hunk 3]
```diff
@@ -190,12 +190,12 @@ def share_params(self, base_class, shared_level, resume=False):
...
         Share the parameters of self to the base_class with shared_level during multitask training.
         If not start from checkpoint (resume is False),
-        some seperated parameters (e.g. mean and stddev) will be re-calculated across different classes.
+        some separated parameters (e.g. mean and stddev) will be re-calculated across different classes.
         """
         assert (
             self.__class__ == base_class.__class__
         ), "Only descriptors of the same type can share params!"
-        # For SeR descriptors, the user-defined share-level
+        # For set descriptors, the user-defined share-level
```
Comment (wanghan-iapcm) [inline deepmd/pt/model/descriptor/se_r.py:198]:
```suggestion
        # For SeR descriptors, the user-defined share-level
```
to be reverted.

**Review by wanghan-iapcm — CHANGES_REQUESTED** [review]

**Review by wanghan-iapcm — APPROVED** [review]

**Pull request merged**